unstructured/ingest/connector/google_drive.py

```diff
@@ -76,7 +76,7 @@ class SimpleGoogleDriveConfig(BaseConnectorConfig):
     recursive: bool = False
 
     def __post_init__(self):
-        if self.extension and self.extension not in EXT_TO_FILETYPE.keys():
+        if self.extension and self.extension not in EXT_TO_FILETYPE:
```

Comment:
Please revert the change here and in `onedrive.py` since that's not related to the EPUB work.